lib/ex_web3_ec_recover/recover_signature.ex

```diff
@@ -11,6 +11,9 @@ defmodule ExWeb3EcRecover.RecoverSignature do
   alias ExWeb3EcRecover.SignedType
   alias ExWeb3EcRecover.SignedType.Message
 
+  defguard is_valid_signature(signature)
```

Comment:
Can this be renamed?

```suggestion
  defguard is_valid_signature?(signature)
```

Reply:
I am more than happy with this change.